enterprise/views.py

```diff
@@ -950,6 +951,14 @@ def get_enterprise_course_enrollment_page(
                 course_start_date = parse(course_run['start']).strftime('%B %d, %Y')
                 now = datetime.datetime.now(pytz.UTC)
                 course_in_future = parse(course_run['start']) > now
+            if course_run['start'] and course_run['end'] and course_run['weeks_to_complete']:
+                course_end_date = parse(course_run['end']).strftime('%B %d, %Y')
+                weeks_to_complete = course_run['weeks_to_complete']
+                course_duration = _('{num_weeks} weeks, Starting on {start_date} and ending at {end_date}').format(
+                    num_weeks=weeks_to_complete,
```

Comment:
I think we can format `weeks_to_complete` so that it handles singular and plural weeks.
```
total_weeks=ungettext(
    '{} week',
    '{} weeks',
    weeks_to_complete
).format(weeks_to_complete) if weeks_to_complete else ''
```

Reply:
Done